TESTING.md

```diff
@@ -2,6 +2,15 @@
 
 ## subscription-manager
 
+First, install the tests dependencies from [`test-requirements.txt`](test-requirements.txt). Make sure your virtual environment is activated.
+
+```bash
+pip install -r test-requirements.txt
+```
+
+***NOTE:** Don’t use `./setup.py test`; the command will most probably fail. Running tests this way is deprecated and has been even removed from the most recent versions of setuptools.*
+
+
```

Comment:
I don't think this is necessary, we're explicitly saying you should be running `pytest` just below.

Reply:
Removed.

---

It may be still useful to mention that a standard way of running tests doesn’t work, just as anybody can be tempted to try runnin `make test` in a project with a Makefile. `setup.py` contains `test_require`, which is only used by `./setup.py test`, creating an impression this way of running tests is supported, or at least that it’s the way to install the test dependencies.

That’s how I found it, because `test-requirements.txt` is not mentioned in this guide and I didn’t notice the test requirements file. Searching for the name of the missing package lead me to the list in `setup.py` and from there to `tests_require`. 

Adding `pip install -r test-requirements.txt` to the guide can hopefully mitigate this. If we later remove `test_require` entirely, the space for confusion will disappear.

Reply:
The tool is just old and rusty on too many places :)